adr/webui.md

```
@@ -0,0 +1,20 @@
+# A couple thoughts:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)